src/js/LayoutManager.js

```diff
@@ -505,6 +505,21 @@ lm.utils.copy( lm.LayoutManager.prototype, {
 		return dragSource;
 	},
 
+
+	/**
+	 * Create a new item in a dragging state, given a starting mouse event to act as the initial position
+	 *
+	 * @param   {Object|Function} itemConfig for the new item to be created, or a function which will provide it
+	 * @param   {MouseEvent} event used as the starting position for the dragProxy
+	 *
+	 * @returns {void}
+	 */
+	createDragSourceFromEvent: function( itemConfig, event ) {
+		this.config.settings.constrainDragToContainer = false;
+		new lm.controls.DragSourceFromEvent(itemConfig, this, event );
```

Comment:
I guess createDragSource tracks so it can destroy in the rare case that the layout is destroyed mid drag. Not sure if that's actually possible, but sure, I can just re-use that logic.

Reply:
Actually, I retract that. There's no reason to hold onto a reference to this, just remove it later.

Looks like dragSource does it because it's creating one from an element that still exists. This is transient with the event.

Reply:
I could return it in case the app wants to abort it for some reason.